experimental/addons/redis-exporter/README.md

```diff
@@ -0,0 +1,73 @@
+# Redis Exporter
+
+Prometheus Redis metrics exporter.
+
+
+
+## Use case example
+
+* Work as a trait.
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - name: redis
+      type: helm
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "16.8.5"
+        values:
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+      traits:
+        - properties:
+          address: <host>:<port>
+          password: <password>
+          disableAnnotation: false
+          name: redis-server-exporter
+          type: redis-exporter
+```
+
+* Work as a component.
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: redis
+  namespace: default
+spec:
+  components:
+    - name: redis
+      type: helm
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "16.8.5"
+        values:
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+    - name: redis-exporter
+      properties:
+        address: <host>:<port>
+        password: <password>
```

Comment:
Explicitly defining passwords within an application may not be the most secure approach. Instead, it may be more advisable to reference an existing secret for enhanced security and ease of management

Reply:
ok, I will fix it.

Reply:
Now the password and username are reference to an existing secret which need to be created before. Because Redis created by helm will also create a Secret, but I can't ensure the secret will be there when Redis is created in other way.